docs/includes/preface.md

```diff
@@ -74,39 +70,49 @@ ln.finish()  # mark the run as finished & save a report for the current notebook
 :::{tab-item} R
 :sync: r
 
-Install the `laminr` R and the `lamindb` Python packages.
+For setup, install the `laminr` and `lamindb` packages and connect to a LaminDB instance.
 
 ```R
 install.packages("laminr", dependencies = TRUE)  # install the laminr package from CRAN
 laminr::install_lamindb(extra_packages = c("bionty"))  # install lamindb & bionty for use via reticulate
-```
-
-Connect to a LaminDB instance.
-
-```R
+laminr::lamin_login()  # <-- you can skip this for public, local, and self-hosted instances
 laminr::lamin_connect("<account>/<instance>")  # <-- replace with your instance
 ```
 
-Access an input dataset and save an output dataset.
+In your R session, you access an input dataset and save an output dataset.
 
 ```R
 library(laminr)
-ln <- import_module("lamindb")  # instantiate the central `ln` object of the API
+ln <- import_module("lamindb")  # instantiate the central object of the API
 
-ln$track()  # track a run of your notebook or script
-artifact <- ln$Artifact$get("3TNCsZZcnIBv2WGb0001")  # get an artifact by uid
-filepath <- artifact$cache()  # cache the artifact on disk
+# Access inputs
 
-# do your work
+ln$track()  # track your run of a notebook or script
+artifact <- ln$Artifact$using("laminlabs/cellxgene")$get("7dVluLROpalzEh8m")  # https://lamin.ai/laminlabs/cellxgene/artifact/7dVluLROpalzEh8m
+adata <- artifact$load()  # load the artifact into memory or sync to cache via filepath <- artifact$cache()
+
+# Your transformation
+
+library(Seurat)  # find marker genes with Seurat
+seurat_obj <- CreateSeuratObject(counts = as(Matrix::t(adata$X), "CsparseMatrix"), meta.data = adata$obs)
+seurat_obj[["RNA"]] <- AddMetaData(GetAssay(seurat_obj), adata$var)
+Idents(seurat_obj) <- "cell_type"
+seurat_obj <- NormalizeData(seurat_obj)
+markers <- FindAllMarkers(seurat_obj, features = Features(seurat_obj)[1:100])
+seurat_path <- tempfile(fileext = ".rds")
+saveRDS(seurat_obj, seurat_path)
+
+# Save outputs
 
-ln$Artifact("./my_dataset.csv", key="my_results/my_dataset.csv").save()  # save a file
-ln$finish()  # mark the run finished
+ln$Artifact(seurat_path, key = "my-datasets/my-seurat-object.rds")$save()
+ln$Artifact$from_df(markers, key = "my-datasets/my-markers.parquet")$save()
+ln$finish()  # finish the run, save source code & run report
 ```
 
-If you did _not_ use RStudio's notebook mode, create an html export externally and run.
+If you did _not_ use RStudio's notebook mode, create an html export and run the following on the CLI.
```

Comment:
Nit: Not sure whether it's correct to say `on the CLI` because you `use` or `run` the CLI. 

I think any of these would be more correct:

"in the terminal"
"in the command line"
"from the terminal"
"using the CLI"
"via the command line"

Reply:
Agreed! Will change